setup.py

```diff
@@ -9,7 +9,7 @@
 # Installed by pip install pdr-backend
 # or pip install -e .
 install_requirements = [
-    "black==24.4.2",
+    "black==24.2.0",
```

Comment:
please restore the black version

Reply:
this is due to poor merges/etc from main.

none of this is modified in the cherry-picked PR => https://github.com/oceanprotocol/pdr-backend/pull/1423